README.md

```diff
@@ -33,10 +33,12 @@ menu = ScriptsMenu(title="Scripts",
 menu.add_script(parent=menu,
                 title="Script A",
                 command="print('A')",
+                sourcetype='',
```

Comment:
Well spotted, this should've been fixed in the README. Though I don't believe an empty string is the fix we're looking for. This should be `sourcetype="python"`